src/Microsoft.DotNet.Interactive.Http/HttpKernel.cs

```diff
@@ -58,6 +56,12 @@ public class HttpKernel :
         RegisterForDisposal(_client);
     }
 
+    public Task HandleAsync(ClearValues command, KernelInvocationContext context)
```

Comment:
Could we implement this using explicit interface implementation and avoid the public API similar to what we do for the other commands below?